{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/settings.py

```diff
@@ -127,3 +127,42 @@
     },
 }
 {%- endif %}
+
+
+# Logging
+LOGGING = {
+    "version": 1,
+    "disable_existing_loggers": False,
+    "formatters": {
```

Comment:
Create a json formatter to run on prod. Use a new env variable called LOG_FORMATTER. The default log formatter should be JSON.

Also, add another a plaintext formatter to be used locally.